src/components/MenuItem.tsx

```diff
@@ -253,6 +253,9 @@ type MenuItemBaseProps = {
 
     /** Handles what to do when the item is focused */
     onFocus?: () => void;
+
+    /** The text to display under the title */
+    underTitle?: string;
```

Comment:
We don't need this as we already have a `furtherDetails` prop, which does the same